README.md

```diff
@@ -927,6 +927,115 @@ constructor(props) {
 
 `componentWillUnmount()` - is invoked immediately before a component is unmounted and destroyed. Useful for resource cleanup.
 
+### Vue.js
+
+#### beforeCreate
+
+Called synchronously immediately after the instance has been initialized, before data observation and event/watcher setup.
+
+```javascript
+new Vue({
+  beforeCreate: function () {
+    // `this` points to the vm instance
+    console.log('this method called before instance created')
+  }
+})
+```
+
+#### created
+
+Called synchronously after the instance is created. At this stage, the instance has finished processing the options which means the following have been set up: data observation, computed properties, methods, watch/event callbacks. However, the mounting phase has not been started, and the $el property will not be available yet.
+
+```javascript
+new Vue({
+  created: function () {
+    // `this` points to the vm instance
+    console.log('this method called after instance created')
+  }
+})
+```
+
+#### beforeMount
+
+Called right before the mounting begins: the render function is about to be called for the first time.
+
+__This hook is not called during server-side rendering.__
+
+```javascript
+new Vue({
+  beforeMount: function () {
+    // `this` points to the vm instance
+    console.log('this method called before mounting an instance')
+  }
+})
+```
+
+#### mounted
+
+Called after the instance has been mounted, where `el` is replaced by the newly created `vm.$el`. If the root instance is mounted to an in-document element, vm.$el will also be in-document when `mounted` is called.
```

Comment:
What is `el`? That's not quite clear for non-Vue devs.

Reply:
should i just link the `el` to https://vuejs.org/v2/api/#el ?

Reply:
Exactly, I also forgot what `el` is until I read the description.